app/src/main/java/org/mozilla/fenix/settings/creditcards/controller/CreditCardEditorController.kt

```diff
@@ -52,36 +55,57 @@ interface CreditCardEditorController {
  * @param ioDispatcher [CoroutineDispatcher] used for executing async tasks. Defaults to [Dispatchers.IO].
  */
 class DefaultCreditCardEditorController(
+    context: Context,
     private val storage: AutofillCreditCardsAddressesStorage,
     private val lifecycleScope: CoroutineScope,
     private val navController: NavController,
     private val ioDispatcher: CoroutineDispatcher = Dispatchers.IO
 ) : CreditCardEditorController {
 
+    private val metrics = context.components.analytics.metrics
+
     override fun handleCancelButtonClicked() {
         navController.popBackStack()
     }
 
+    /**
+     * Deletes an existing credit card.
+     *
+     * @param guid the id of the credit card to delete.
+     */
```

Comment:
I think in general since these are override functions we don't need to add kDoc for them since they belong in their respective interfaces instead. Having them here would be reductant. I looked through our codebase and don't see any instances where we needed to add a kDoc for override functions.